runners/local/runner.go

```diff
@@ -142,6 +143,14 @@ func NewRunnerLocal(settings RunnerLocalSettings) *RunnerLocal {
 	nullWrite := settings.DstConnString == "/dev/null"
 	if nullWrite {
 		r.dst = connectorNull.NewNullConnector(destinationName)
+	} else if settings.DstType == "CosmosDB" {
+		connSettings := connectorCosmos.ConnectorSettings{ConnectionString: settings.DstConnString}
+		if settings.LoadLevel != "" {
+			btc := getBaseThreadCount(settings.LoadLevel)
+			connSettings.NumParallelWriters = btc * 2 // double the base thread count to have more writers than readers (accounting for latency)
+		}
+		r.dst = connectorCosmos.NewCosmosConnector(destinationName, connSettings)
```

Comment:
I suggest you add the writer specific Cosmos connector settings here, such as WriterMaxBatchSize and NumParallelWriters. May as well include setting of PingTimeout and ServerConnectTimeout.